docs/platform/what-is-tizen/versions/tizen-6-0-m2.md

```diff
@@ -0,0 +1,103 @@
+# Tizen 6.0
+
+The second milestone (M2) release of Tizen 6.0 was announced in October 2020.
+Tizen is an open-source operating system (OS) maintained by Samsung and is developed and optimized for smart devices since 2012.
+
+The key features of Tizen 6.0 are the enhancement of the development environment, core, and AI components.
+
+## Enhancement of Development Environment and Core Component
```

Comment:
Can you change it to: 

## Enhancement of development environment and core component